src/Components/Footer/Footer.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { View } from 'react-native';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { removeSelected } from '../../Actions';
+import DeleteSelectedButton from '../Buttons/DeleteSelectedButton';
+
+import Styles from './Styles';
+
+const Footer = ({ dispatchRemoveSelected }) => {
+  const { footerStyle } = Styles;
+  return (
+    <View style={footerStyle}>
+      <DeleteSelectedButton onPress={() => dispatchRemoveSelected()} />
```

Comment:
There is no need of creating the arrow function, you can pass `dispatchRemoveSelected` directly.

Be aware that since you connected this component to redux it is not reusable.

Reply:
I corrected 'dispatchRemoveSelected' as mentioned.
About the reusability of the component, as it is the footer, I don't need to reuse it.